core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/ConcentrationUnit.java

```diff
@@ -0,0 +1,27 @@
+package uk.ac.bbsrc.tgac.miso.core.data;
+
+import java.util.Arrays;
+import java.util.List;
+
+public enum ConcentrationUnit {
+
+  NANOGRAMS_PER_MICROLITRE("ng/&#181;L", UnitType.SAMPLE, UnitType.LIBRARY, UnitType.DILUTION, UnitType.POOL),
+  NANOMOLAR("nM", UnitType.SAMPLE, UnitType.LIBRARY, UnitType.DILUTION, UnitType.POOL);
+
+  private final String units;
+  private final List<UnitType> unitTypes;
```

Comment:
I think it's unlikely that we'll ever want different options for volume or concentration units on different types, so this (and resulting filtering) probably isn't necessary